### ISSUE TITLE
Installation Instructions

### ISSUE THREAD

**Issue by jefflei** [taxcloud/Magento#1]
To Install:

Create a directory in app/code/community called Ia\TaxCloud.

Copy the directories: Helper, Model, sql and etc to this directory.

Rename the attached IA_TaxCloud.txt to IA_TaxCloud.xml and copy into the app\etc\Modules directory

[IA_TaxCloud.txt](https://github.com/taxcloud/Magento/files/237784/IA_TaxCloud.txt)

Log out of Magento Admin, then log in again. You should see TaxCloud under the System/Configuration.

Thanks to all who made this possible.

**Comment by NoMessProducts** [comment]
Hi Jefflei,

I followed your instructions but now my system/configuration is blank.  Any clue on what I may have missed?

Thanks for the help

**Comment by jefflei** [comment]
Please browse to /app/code/community/Ia/TaxCloud and you should see the following directories:

etc, Helper, Model and sql.

By the way Ia is typed as capital i and then lower case a.

**Comment by cmilburndesign** [comment]
Jefflei,

I followed your installation directions, but I'm not seeing "TaxCloud" withing the System -> Configuration page. Logging out, clearing cache and disabling/enabling the module still do not resolve the missing settings. Can you provide any assistance? (We're running Magento CE 1.8.1)

**Comment by jefflei** [comment]
Did you log out and log back in again?

**Comment by cmilburndesign** [comment]
Yes, logging out and back in did not trigger it to show on the configuration page.

The module does show that it's enabled in the System -> Configuration-> Advanced section. Not sure what else to try.


-------------------------------
Sent from my iPhone 6
Cleveland, OH

**Comment by cmilburndesign** [comment]
@jefflei, still no luck getting TaxCloud to show up in our Magento Admin (System -> Configuration). What should we be looking for? "TaxCloud" is not listed in any of the lefthand config lists. Can you provide a screenshot of what we _should_ see? Is there any way to find out if our TaxCloud code is misconfigured? 

Our client is anxiously looking to have this implemented on their staging site. Are there any dependencies that we may be missing?

**Comment by jefflei** [comment]
Hi,

I think this is the file which goes into app/etc/modules.

I am also attaching the zip file I used.  I am not a programmer, but
managed to get this working...somehow.

Sorry I cannot be of any additional help.

Jeff


> @jefflei, still no luck getting TaxCloud to show up in our Magento Admin
> (System -> Configuration). What should we be looking for? "TaxCloud" is
> not listed in any of the lefthand config lists. Can you provide a
> screenshot of what we _should_ see? Is there any way to find out if our
> TaxCloud code is misconfigured?
>
> Our client is anxiously looking to have this implemented on their staging
> site. Are there any dependencies that we may be missing?
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/taxcloud/Magento/issues/1#issuecomment-230565710


-- 
Jeffrey Chalmers, CFO
jeff.chalmers@sheeninc.com
408-833-5329

**Comment by cmilburndesign** [comment]
@taxcloud can you provide us with Magento installation instructions for TaxCloud? I've followed the provided instructions above, but the TaxCloud config does not show up within our Magento 1.8.1 admin configurations.

**Comment by sanil637** [comment]
@cmilburndesign  Hi, Create a directory in app/code/community called Ia\TaxCloud.
(Ia -  Capital i and small a)
Copy the directories: Helper, Model, sql and etc to this directory.
[Ia_TaxCloud.txt](https://github.com/taxcloud/Magento/files/351580/Ia_TaxCloud.txt)

download the attached file and rename it into Ia_TaxCloud.xml and copy to app\etc\Modules directory

try with file i have attached  it will work

**Comment by cmilburndesign** [comment]
@taxcloud, thank you! The issue had to do with the above instructions naming the xml file to:
_IA_TaxCloud.txt_ (Incorrect capital "A")

Instead of:
_Ia_TaxCloud.txt_

**Comment by Premmy** [comment]
I followed the instructions and subsequent responses and able to see the Tax Cloud menu in Magento 1.9.3 CE configuration. I attempted a few sales transactions but did not get any tax responses.  Besides enabling and inputting the keys, are any additional configurations required? 

to recap:

1. Create a directory in app/code/community called I**a**\TaxCloud.
2. Copy the directories: Helper, Model, sql and etc to this directory.
3. Rename the attached IA_TaxCloud.txt to I**a**_TaxCloud.xml
4. Within the XML change the "I**A**_TaxCloud" to "I**a**_TaxCloud"
5. Copy into the app\etc\Modules directory
6. Log out of Magento Admin, then log in again. 
7. You should see TaxCloud under the System/Configuration.

It looks like the lowercase "a" needs to be updated in 3 places. Can the original post should be updated with the correct file and instructions?

**Comment by sanganagouda1** [comment]
Hi,
I am working in developing the taxcloud plugin for magento 2 ..But Lookup API response sending the tax value 0 so please do the needful

**Comment by taxcloud** [comment]
Note: you must log into your TaxCloud account to configure which states you want to collect sales tax in. ;)
